openvdb/cmd/openvdb_print.cc

```diff
@@ -141,7 +245,18 @@ printLongListing(const StringVec& filenames)
                 grid->print(std::cout, /*verboseLevel=*/11);
                 firstGrid = false;
             }
+#if OPENVDB_ABI_VERSION_NUMBER >= 6
+            // Check for grid type, inspect points if this is a point grid
+            const openvdb::GridBase::ConstPtr grid = *it;
```

Comment:
I'd move this to the start of the loop so the shared pointer is only copied once, immediately followed by a check